crates/lib/src/qpu/client.rs

```diff
@@ -193,7 +193,7 @@ pub enum GrpcClientError {
     GrpcError(#[from] GrpcError),
 }
 
-/// Errors that may occur while trying to use a [`OpenAPI`] client
+/// Errors that may occur while trying to use an ``OpenAPI`` client
```

Comment:
Same note as `gRPC` -- this could also be added to `clippy.toml`

Reply:
Switched to single backticks for these.